kr/debug/profiling.md

```diff
@@ -33,13 +33,13 @@ PMSP는 GDB를 사용하여 stack trace를 수집합니다.
 사용 예제:
 
 ```bash
-./poor-mans-profiler.sh --elf=build/px4fmu-v4_default/src/firmware/nuttx/firmware_nuttx --nsamples=30000
+./poor-mans-profiler.sh --elf=build/px4_fmu-v4_default/src/firmware/nuttx/firmware_nuttx --nsamples=30000
```

Comment:
ditto - is that full path after --elf= correct?

Reply:
No, that's been wrong for a long time.

Reply:
./build/px4_fmu-v4_default/px4_fmu-v4_default.elf